diff --git a/node_modules/tsx-dom-types/dist/HTMLAttributes.d.ts b/node_modules/tsx-dom-types/dist/HTMLAttributes.d.ts
index 837d2a6..670a138 100644
--- a/node_modules/tsx-dom-types/dist/HTMLAttributes.d.ts
+++ b/node_modules/tsx-dom-types/dist/HTMLAttributes.d.ts
@@ -51,7 +51,7 @@ export interface HTMLAttributes {
     disableRemotePlayback?: boolean;
     download?: string;
     decoding?: "sync" | "async" | "auto";
-    draggable?: boolean;
+    draggable?: "true" | "false";
     enctype?: string;
     enterKeyHint?: "enter" | "done" | "go" | "next" | "previous" | "search" | "send";
     form?: string;
